docs/core-manage-asdf-vm.md

```diff
@@ -184,6 +192,36 @@ Add the following to `~/.zshrc`:
 
 ?> Completions will need to be [configured as per Homebrew's instructions](https://docs.brew.sh/Shell-Completion#configuring-completions-in-zsh).
 
+### --Linux,Bash,Pacman--
+
+Add the following to `~/.bashrc`:
+
+```shell
+. /opt/asdf-vm/asdf.sh
+```
+
+?> Completions are automatically configured on installation by the aur package.
+
+### --Linux,Fish,Pacman--
+
+Add the following to `~/.config/fish/config.fish`:
+
+```shell
+source /opt/asdf-vm/asdf.sh
+```
+
+!> Completions may not be automatically configured for Fish, please check the [aur package comments](https://aur.archlinux.org/packages/asdf-vm/) before adding completions manually.
+
+### --Linux,ZSH,Pacman--
+
+Add the following to `~/.zshrc`:
+
+```shell
+. /opt/asdf-vm/asdf.sh
+```
+
+?> Completions are automatically configured on installation by the aur package.
```

Comment:
The completions are put in a location where ZSH can see them, but [ZSH doesn't autocomplete by default](https://wiki.archlinux.org/index.php/zsh#Command_completion).